plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java

```diff
@@ -125,10 +129,48 @@ public HiveHadoop getHiveHadoop()
         return hiveHadoop;
     }
 
+    public void copyResourcesToDataLake(String resourcePath, String target)
+    {
+        try {
+            List<ResourceInfo> resourceInfoList = ClassPath.from(HiveMinioDataLake.class.getClassLoader())
+                    .getResources()
+                    .stream()
+                    .filter(resourceInfo -> resourceInfo.getResourceName().startsWith(resourcePath))
+                    .collect(Collectors.toList());
+            AmazonS3 s3Client = getS3Client();
+            for (ResourceInfo resourceInfo : resourceInfoList) {
+                String fileName = resourceInfo.getResourceName().replaceFirst("^" + Pattern.quote(resourcePath), quoteReplacement(target));
+                s3Client.putObject(bucketName, fileName, Files.readFile(resourceInfo.asByteSource().openStream()));
+            }
+        }
+        catch (Exception e) {
+            String message = String.format("Could not copy resources %s from classpath", resourcePath);
+            logger.warn(e, message);
+            throw new RuntimeException(message, e);
+        }
+    }
+
     @Override
     public void close()
             throws IOException
     {
         stop();
     }
+
+    private AmazonS3 getS3Client()
+    {
+        return this.s3Client.orElseGet(() -> {
+            this.s3Client = Optional.of(
+                    AmazonS3ClientBuilder
+                            .standard()
+                            .withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration(
+                                    "http://localhost:" + minio.getMinioApiEndpoint().getPort(),
```

Comment:
nit: Can we have an API that directly provides the Minio Endpoint in localhost.

Reply:
added 2 private methods. For Endpoint and Credentials